client/src/views/Comparison/Comparison.vue

```diff
@@ -92,18 +82,7 @@
         this.$route.params.model_id.split(',').forEach(this.setSelectedModels);
       }
 
-      const selectedModels = this.getModelsList.filter(model => this.getSelectedModelIds.map(Number).includes(model.id));
-      // HACK: Adding the overlapping elements to the model graph information
-      let nodes = [];
-      selectedModels.forEach(model => {
-        if (model.name === 'SimpleSIR_metadata') {
-          nodes = Object.keys(MODEL_COMPARISON).map(node => ({ id: node }));
-        } else if (model.name === 'SimpleChime+') {
-          nodes = Object.values(MODEL_COMPARISON).map(node => ({ id: node }));
-        }
-        model.modelGraph[0].overlappingElements = { nodes, edges: [] };
-      });
-      return selectedModels;
+      return this.getModelsList.filter(model => this.getSelectedModelIds.map(Number).includes(model.id));
```

Comment:
You don't need to convert to `Number`, `includes()` works on string with Numbers. 
i.e.
 ```JS
'1,2,3'.includes(3); // true
```